ssl/ssl_rsa.c

```diff
@@ -916,6 +916,9 @@ int SSL_CTX_use_serverinfo_file(SSL_CTX *ctx, const char *file)
     char *header = NULL;
     char namePrefix1[] = "SERVERINFO FOR ";
     char namePrefix2[] = "SERVERINFOV2 FOR ";
+    unsigned int namePrefix1_len;
```

Comment:
you can just use `sizeof(namePrefix1) - 1` here and avoid more `strlen`s

Reply:
the variables should be "static"